### DISCUSSION TITLE
Multilingualization of messages

### DISCUSSION THREAD

**Discussion by mtshiba** [erg-lang/erg#40] [Ideas]
I am planning to multilingualize error messages (currently only English, Japanese, Simplified Chinese and Traditional Chinese are supported).

This will be documented so that people who are not familiar with Rust can participate.

Is there anyone who would like to help?

**Comment by C-BJ** [comment]
I can help translate Simplified Chinese and Traditional Chinese

**Comment by mtshiba** [comment]
Thank you! I will prepare instructions.

**Comment by mtshiba** [comment]
I have prepared a multilingualization! Please see [/doc/EN/dev_guide/i18n_messages.md](https://github.com/erg-lang/erg/blob/main/doc/EN/dev_guide/i18n_messages.md) for details.
